protocol/avdl/keybase1/simple_fs.avdl

```diff
@@ -264,59 +264,59 @@ protocol SimpleFS {
   /**
    Begin copy of file or directory 
    */
-  void simpleFSCopy(OpID opID, Path src, Path dest);
+  void simpleFSCopy(OpID opID, Path src, Path dest, union{ null, TLFIdentifyBehavior } identifyBehavior);
 
 
   /**
    Make a symlink of file or directory
    */
-  void simpleFSSymlink(string target, Path link);
+  void simpleFSSymlink(string target, Path link, union{ null, TLFIdentifyBehavior } identifyBehavior);
 
   /**
    Begin recursive copy of directory 
    */
-  void simpleFSCopyRecursive(OpID opID, Path src, Path dest);
+  void simpleFSCopyRecursive(OpID opID, Path src, Path dest, union{ null, TLFIdentifyBehavior } identifyBehavior);
 
   /**
    Begin move of file or directory, from/to KBFS only
    */
-  void simpleFSMove(OpID opID, Path src, Path dest);
+  void simpleFSMove(OpID opID, Path src, Path dest, union{ null, TLFIdentifyBehavior } identifyBehavior);
 
   /**
    Rename file or directory, KBFS side only
    */
-  void simpleFSRename(Path src, Path dest);
+  void simpleFSRename(Path src, Path dest, union{ null, TLFIdentifyBehavior } identifyBehavior);
 
   /**
    Create/open a file and leave it open
    or create a directory
    Files must be closed afterwards.
    */
-  void simpleFSOpen(OpID opID, Path dest, OpenFlags flags);
+  void simpleFSOpen(OpID opID, Path dest, OpenFlags flags, union{ null, TLFIdentifyBehavior } identifyBehavior);
 
   /**
    Set/clear file bits - only executable for now
   */
-  void simpleFSSetStat(Path dest, DirentType flag);
+  void simpleFSSetStat(Path dest, DirentType flag, union{ null, TLFIdentifyBehavior } identifyBehavior);
 
   /**
    Read (possibly partial) contents of open file,
    up to the amount specified by size.
    Repeat until zero bytes are returned or error.
    If size is zero, read an arbitrary amount.
    */
-  FileContent simpleFSRead(OpID opID, long offset, int size);
+  FileContent simpleFSRead(OpID opID, long offset, int size, union{ null, TLFIdentifyBehavior } identifyBehavior);
```

Comment:
Read and Write don't need this param, because it should already have been triggered by Open I think.  Seems like a good rule of thumb is that it should go along with any Path that can be a KBFS path.

You've already done the work here to plumb it through as a separate param, which is fine with me, but perhaps in the future, or if you like the idea a lot better, we could make a `KbfsPath` type in the avdl that contains both a string and the identify behavior?  That would probably be a significant code change though, since it's currently just a bare string everywhere.

Reply:
That's a great idea! I don't think it'd be as bad as my current plumbing. The identify behavior is optional, so we'll only need to add that in GUI. Will try to do that. Thanks!

Reply:
Ah I see what you mean here. It's a variant and the kbfs case is just a string. Will try and see how bad it is